src/manipulation.js

```diff
@@ -162,7 +162,9 @@ function domManip( collection, args, callback, ignored ) {
 							// Optional AJAX dependency, but won't run scripts if not present
 							if ( jQuery._evalUrl && !node.noModule ) {
 								jQuery._evalUrl( node.src, {
-									nonce: node.nonce || node.getAttribute( "nonce" )
+									nonce: node.nonce || node.getAttribute( "nonce" ),
+									crossOrigin: node.crossOrigin ||
```

Comment:
I don't think you need to read both the attribute & the property. Support for both seems similar:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script
https://developer.mozilla.org/en-US/docs/Web/API/HTMLScriptElement
(look at the "Browser compatibility" tables below).

The reason why for `nonce` we needed both the property & the attribute is that in Chrome (& latest standards) `getAttribute( "nonce" )` doesn't return an up to date `nonce` value:
https://github.com/whatwg/html/issues/2369
https://html.spec.whatwg.org/#nonce-attributes
but, at the same time, the `nonce` property is not supported in Firefox & Edge so we need the `getAttribute` fallback for them.

I'd just leave `crossOrigin: node.crossOrigin` here. But please make sure the tests still pass.